tests/jballerina-debugger-integration-test/src/test/resources/project-based-tests/basic-project/src/hello-world/tests/hello_world_test.bal

```diff
@@ -33,5 +33,9 @@ public function mockPrint((any|error)... s) {
 function testFunc() {
     // Invoking the main function
     main();
-    test:assertEquals(outputs[0], "Hello, World!");
+    if (outputs[0] is anydata) {
```

Comment:
Should we ask the user to handle this? Is there a possibility to hide this detail from the user and handle it internally maybe within the `assertEquals` function?

Reply:
That is like going back to the previous implementation right?

Reply:
Did you mean to allow user to pass "any" type into the function and check it if it qualifies as "anydata" and do a value check?
The problem is, this check happens at runtime. If user passes a value not qualifying as "anydata", still the function would allow it and user would expect value equality. But, he would get notified about the failure only at runtime.